src/main/java/com/adyen/Config.java

```diff
@@ -143,7 +145,7 @@ public void setHmacKey(String hmacKey) {
     }
 
     public String getCheckoutEndpoint() {
-        if (checkoutEndpoint == null || checkoutEndpoint.isEmpty()) {
+        if (Objects.isNull(checkoutEndpoint) || checkoutEndpoint.isEmpty()) {
```

Comment:
Hey @smic1909 

I find the Objects.isNull a bit redundant when it's not being used as a predicate in a stream context. It's just object == null on the inside as well. If you agree, could you restore these one back to the regular null check.